scripts/monitoring/monitor_ecs_metrics.py

```diff
@@ -0,0 +1,226 @@
+    "SECRET_KEY": settings.SECRET_KEY,
+    "SENTRY_URL": os.environ.get("SENTRY_URL"),
+    "STATSD_ENDPOINT": os.environ.get("STATSD_ENDPOINT"),
+    "STATSD_PORT": os.environ.get("STATSD_PORT"),
+}
+
+
+def get_boto3_client(resource, aws_keys):
+    import boto3
+    """
+    Returns the boto3 client for a resource in AWS
+    Arguments:
+        resource {str} -- Name of the resource for which client is to be created
+        aws_keys {dict} -- AWS keys which are to be used
+    Returns:
+        Boto3 client object for the resource
+    """
+    try:
+        client = boto3.client(
+            resource,
+            region_name=aws_keys["AWS_REGION"],
+            aws_access_key_id=aws_keys["AWS_ACCESS_KEY_ID"],
+            aws_secret_access_key=aws_keys["AWS_SECRET_ACCESS_KEY"],
+        )
+        return client
+    except Exception as e:
+        print("Error in creating boto3 client: ", str(e))
+        return None
+
+
+def get_cpu_metrics_for_challenge(challenge, cluster_name=COMMON_SETTINGS_DICT["CLUSTER"], range_days=1, period_seconds=300):
+    """
+    Get the CPU Utilization of the worker in the challenge.
+    """
+
+    cloudwatch_client = get_boto3_client("cloudwatch", aws_keys)
+
+    start_time = datetime.utcnow() - timedelta(days=range_days)
+    end_time = datetime.utcnow()
+    queue_name = challenge.queue
+    service_name = "{}_service".format(queue_name)
+
+    response = cloudwatch_client.get_metric_statistics(
+        Namespace="AWS/ECS",
+        MetricName="CPUUtilization",
+        Dimensions=[
+            {"Name": "ClusterName", "Value": cluster_name},
+            {"Name": "ServiceName", "Value": service_name},
+        ],
+        StartTime=start_time,
+        EndTime=end_time,
+        Period=period_seconds,
+        Statistics=["Average", "Maximum", "Minimum"],
+    )
+
+    return response["Datapoints"]
+
+
+def get_memory_metrics_for_challenge(challenge, cluster_name=COMMON_SETTINGS_DICT["CLUSTER"], range_days=1, period_seconds=300):
+    """
+    Get the Memory Utilization of the worker in the challenge.
+    """
+
+    cloudwatch_client = get_boto3_client("cloudwatch", aws_keys)
+
+    start_time = datetime.utcnow() - timedelta(days=range_days)
+    end_time = datetime.utcnow()
+    queue_name = challenge.queue
+    service_name = "{}_service".format(queue_name)
+
+    response = cloudwatch_client.get_metric_statistics(
+        Namespace="AWS/ECS",
+        MetricName="MemoryUtilization",
+        Dimensions=[
+            {"Name": "ClusterName", "Value": cluster_name},
+            {"Name": "ServiceName", "Value": service_name},
+        ],
+        StartTime=start_time,
+        EndTime=end_time,
+        Period=period_seconds,
+        Statistics=["Average", "Maximum", "Minimum"],
+    )
+
+    return response["Datapoints"]
+
+
+def get_storage_metrics_for_challenge(challenge, cluster_name=COMMON_SETTINGS_DICT["CLUSTER"], range_days=1, period_seconds=300):
```

Comment:
Use `evalai-production-cluster` as the default cluster name.

Reply:
For sure.